athena-postgresql/athena-postgresql.yaml

```diff
@@ -0,0 +1,103 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Metadata:
+  'AWS::ServerlessRepo::Application':
+    Name: AthenaPostgresqlConnector
```

Comment:
nit: AthenaPostgreSQLConnector

Reply:
Similar to how it's written everywhere, I'll change this to `AthenaPostGreSqlConnector`

Reply:
mmm.. this is what shown in SAR, so i think we should do it how its officially stylized `PostgreSQL`